README.md

```diff
@@ -122,8 +122,7 @@ Change the label text and add a class on the label:
 f.text_field :name, label: 'Nombre', label_options: { class: 'large' }
 ```
 
-<!-- TODO: Help text option   -->
-<!-- If the help_text option is specified
+If the help_text option is specified
```

Comment:
Yes, but I had already created this PR before this commit.  I ended up having time to add #145 and #146, so I went back and updated the readme.